app/src/services/projects.ts

```diff
@@ -343,158 +339,137 @@ const getPaymentDetails = (
     }))
 }
 
-async function toastedProjectOperation(label: string, fn?: () => Promise<void>) {
```

Comment:
Projects don't come with transaction list toasts anymore?

Reply:
They do, but it's not part of the service anymore since we need to customize steps when creating a data union project. You can find toast logic in `ProjectController.tsx`.